### ISSUE TITLE
[Bug][a11y] Manual sign-in onboarding - Increase touch target for 'Learn more' link

### ISSUE THREAD

**Issue by neha-b2001** [mozilla-mobile/fenix#18713]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
1. Install and open Google Accessibility Scanner
2. Have a clean Firefox profile
3. Upon opening the app, a manual sign-in onboarding tip is seen at the top of the screen
### Expected behavior
No suggestions for changes.
### Actual behavior
Suggestions are made for the **Learn more** link:

**Touch target**
Consider making this clickable item larger.
This item's height is 20dp. Consider making the height of this touch target 48dp or larger.
![image](https://user-images.githubusercontent.com/67039214/113042355-b7e23f80-91b8-11eb-82da-2293142a1a66.png)

### Device information

* Android Device: Redmi Note 8 Pro (Android 10)
* Fenix Version: Firefox Preview Version 1.0.2114

**Comment by lobontiumira** [comment]
This is a duplicate of #14716.
I'll close this.

**Comment by neha-b2001** [comment]
@lobontiumira Hello, actually in issue #14716, the Learn more TextView's size has to be increased in `button_tip_item.xml`, whereas for this issue, the Learn more TextView's size needs to be increased in `onboarding_manual_signin.xml`. Hence I thought of opening a separate issue for this, sorry about that 😅 

Can I resolve both these issues in the same PR (for issue #14716) then?

**Comment by eliserichards** [comment]
Let's keep this issue open for `onboarding_manual_signin.xml`

**Comment by neha-b2001** [comment]
Thank you @eliserichards 👍

**Comment by neha-b2001** [comment]
Hello @eliserichards, upon rebasing with upstream master, I just saw that this onboarding sign-in card has been removed altogether. So, I think this issue can now be closed.

**Comment by eliserichards** [comment]
Closing as no longer valid.
